docs/en/developer.md

```diff
@@ -182,3 +183,24 @@ class TaxonomyDirectory_Controller extends Page_Controller
 
 }
 ```
+
+### Using the provided default controller implementation
+If you want to use the provided directory implementation named `TaxonomyDirectoryController`, more or less based on the previous example, the only thing
+you need to do is enable access to the controller functions using a custom `routes.yml` file
+
+```yaml
+
+---
+Name: directoryroutes
+After: framework/routes#coreroutes
+---
+Director:
+  rules:
+    'code/$ID!': 'TaxonomyDirectoryController'
```

Comment:
Maybe use something other than `code` for the example - in case of copy paste implementation ;) use "tag" or "<your path here>" in the docs. Make it clear a user can implement a path to suit them.

Reply:
this needs changing

Reply:
I can live with it as is if you want to get on and merge @dhensby :+1:

Reply:
I've fixed this when merging manually